### DISCUSSION TITLE
How can I integrate picom with my c++ program to manage windows?

### DISCUSSION THREAD

**Discussion by renjuashokan** [yshui/picom#665] [Q&A]
Hi all, 
I'm new to this composite manager. I have a question, can I integrate this with a c++ project? 
I want to switch multiple windows (like bring one window to the top, hide window) in a visually pleasing way (like slides in power point) 
if yes, any example or documentation is available?

**Comment by n-ovux** [comment]
I think you miss understanding what this project is. It's not a library that you can use, it's a compositor for x11. Hope this helps ^_^!
